finnemit/finnemit.py

```diff
@@ -77,11 +77,18 @@ def get_emissions(infile, outfile=None, fuelin=None, emisin=None):
     # READ in LCT Fuel loading file from prior Texas FINN study
     # This is a secondary fuel loading file for use in US ONLY
     lctfuelin = pkg_resources.resource_filename(
-        "finnemit", "data/land-cover-gm2.csv"
+        "finnemit", "data/LCTFuelLoad_fuel4_revisit20190521.csv"
```

Comment:
@yosukefk please modify the existing `land-cover-gm2.csv` file instead of adding another file. That way we can keep the fuel load file under version control, rather than relying on the file name to indicate when it was last modified.